include/megaapi.h

```diff
@@ -2614,6 +2614,7 @@ class MegaRequest
             TYPE_CHAT_LINK_HANDLE, TYPE_CHAT_LINK_URL, TYPE_SET_PRIVATE_MODE, TYPE_AUTOJOIN_PUBLIC_CHAT,
             TYPE_CATCHUP, TYPE_PUBLIC_LINK_INFORMATION,
             TYPE_GET_BACKGROUND_UPLOAD_URL, TYPE_COMPLETE_BACKGROUND_UPLOAD,
+            TYPE_GET_CLOUDSTORAGEUSED,
```

Comment:
Maybe we should refactor it to `TYPE_GET_CLOUD_STORAGE_USED` or `TYPE_CLOUDSTORAGE_USED` for better readability. Only a suggestion, I'm fine with this one as it is :)

Reply:
updated - also I noticed some tabs instead of spaces while making that update, I've converted those also.